dapps/marketplace/src/pages/identity/EmailAttestation.js

```diff
@@ -164,6 +164,25 @@ class EmailAttestation extends Component {
               e.preventDefault()
               if (this.state.loading) return
               this.setState({ error: false, loading: true })
+
+              const trimmedCode = this.state.code.trim()
+
+              if (trimmedCode.length === 0) {
+                this.setState({
+                  error: 'Verification code is required',
+                  loading: false
+                })
+                return
+              }
+
+              if (trimmedCode.length !== 6 || isNaN(trimmedCode)) {
+                this.setState({
+                  error: 'Verification code is incorrect',
```

Comment:
Perhaps say "Verification code should be a 6 digit number"

Reply:
Done. :+1: